app.js

```diff
@@ -87,14 +95,13 @@ document.querySelector('#book-form').addEventListener('submit', (e) => {
 
   // Validate
   if (titleI === '' || authorI === '') {
-    showAlert('title and author must not be empty', 'success');
+    UI.showAlert('title and author must not be empty', 'success');
   } else {
-    const books = getBooks();
-    books.push({
-      title: titleI,
-      author: authorI,
-    });
-    setBooks(books);
+    const books = Store.getBooks(); // get books from local storage
+    const book = new Book(titleI, authorI); // new instance of Book
+    books.push(book); // push new book into books array
+
+    Store.setBooks(books); // sets new books array in local storage
     clearOut();
     // Reload page
     // eslint-disable-next-line no-restricted-globals
```

Comment:
- [ ] [Optional] you would rather fix the error than disabling this rule, it will help you master best practices and become a pro, this error is very possible to solve it, so I think you can try fixing it